services/webhook/tests/test_ticket_compliance_run.py

```
@@ -2,8 +2,21 @@
 
 from __future__ import annotations
 
+import pytest
+
 import personas.tpm.ticket_compliance_run as run
 
+_OWN_APP_ID = "1"  # matches performed_via_github_app.id in the fixtures below
+
+
+@pytest.fixture(autouse=True)
+def _stub_own_app_id(monkeypatch):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_stub_own_app_id`

Add return type annotation: `None`

(ANN202)